service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -53,10 +55,13 @@
 import java.util.stream.Collectors;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cache.annotation.Cacheable;
+import org.springframework.context.annotation.Lazy;
 import org.springframework.dao.DataAccessException;
 import org.springframework.stereotype.Component;
 
+@Lazy
```

Comment:
Why does this need to be init'ed lazily?

Reply:
Unfortunately, we have circular dependencies. We have the same situation in WSM. It is a specific of how we wire up a flight . I will take a look what I can do.